index.html

```diff
@@ -0,0 +1,522 @@
+        <ul class="top-bar1">
+          <li><a href="#">News</a></li>
+          <li><a href="#">Events</a></li>
+          <li><a href="#">Business</a></li>
+          <li><a href="#">AMAs</a></li>
+          <li><a href="#">Spaces</a></li>
+          <li><a href="#">Terms & Conditions</a></li>
+        </ul>
+        <ul class="top-bar2">
+          <li><a href="#">About</a></li>
+          <li><a href="#">Team</a></li>
+          <li><a href="#">Advertise</a></li>
+          <li><a href="#">Jobs</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </section>
+      <section class="lower-menus">
+        <ul class="low-bar">
+          <li><a href="#">LATEST</a></li>
+          <li><a href="#">HARD FORK</a></li>
+          <li><a href="#">PLUGGED</a></li>
+          <li><a href="#">FUNDAMENTALS</a></li>
+          <li><a href="#">WORK 2030</a></li>
+        </ul>
+        <button class="serch-button">
+          <i class="fas fa-search fa-2x"></i>
+        </button>
+        <button class="hidden-menu">
+          <i class="fas fa-bars fa-2x"></i>
+        </button>
+      </section>
+      <section class="nav-message">
+        <a href="#">Did you get your tickets to Hard Fork Summit yet? →</a>
+      </section>
+    </nav>
+  </header>
+  <section class="main-articles">
+      <section class="article one">
+        <header>
+          <section class="cover-logo">
+            <img src="images/tnwlogo.png" alt="">
+          </section>
+        </header>
+        <footer>
+          <span class="genre">TECH</span>
+          <h2>Saturn declared most moonfluential planet after follower count reaches 82</h2>
+          by <a href="#">Tristan Greene</a> <i class="far fa-clock"></i> 1 hour ago <i class="fas fa-share-alt"></i>
+        </footer>
+      </section>
+    <section class="article two">
+      <header>
+
+      </header>
+
+      <footer>
+        <span class="genre">TECH</span>
+        <h2>Everything we learned today about the PlayStation 5</h2>
+        by <a href="#">Rachel Kaser</a> <i class="far fa-clock"></i> 2 hours ago <i class="fas fa-share-alt"></i>
+      </footer>
+    </section>
+    <section class="article three">
+      <header>
+
+      </header>
+      <span class="genre">GEAR</span>
+      <h2>The Tap Strap wearable keyboard gets better with airmouse gestures</h2>
+      <footer>
+        by <a href="#">Napier Lopez</a> <i class="far fa-clock"></i> 3 hours ago <i class="fas fa-share-alt"></i>
+      </footer>
+    </section>
+  </section>
+  <section class="latest-news">
+    <header>
+      <h1><a href="#">Latest news <i class="fas fa-chevron-right"></i></a></h1>
+    </header>
+    <section class="news">
+        <img src="images/default.png" alt="default image">
+        <section class="news-content">
+          <h3><a href="#">Adobe cancels all Venezuelan accounts, highlighting problems with subscription apps</a></h3>
+          <span>Napier Lopez - 1 hour ago</span>
+        </section>
```

Comment:
- [ ] Passing your html document through the [validator](https://validator.w3.org), there are errors about missing headers - h2-h6 and this is because, you have used ```<section> tag``` as a sub-section. For every section tag, there should be header tag, so the validator is expecting h2-h6 tags under each of those sections you created. 
Thus, replace every subsection implemented with a ```<section> tag``` with a ```<div> tag```
Use the **```<section> tag```** for only the main section of the doc